contracts/payloads/IGP86/PayloadIGP86.sol

```diff
@@ -0,0 +1,568 @@
+        updateDexBaseLimits(19, 7_500_000, 0); // deUSD-USDC DEX (Smart Collateral Only)
+        updateDexBaseLimits(20, 7_500_000, 0); // USR-USDC DEX (Smart Collateral Only)
+//        updateDexBaseLimits(21, 0, 0);                    // Skip DEX 21
+        updateDexBaseLimits(22, 14_000_000, 10_000_000); // cbBTC-USDT DEX (Smart Collateral & Smart Debt)
+        updateDexBaseLimits(23, 7_500_000, 0);  // USD0-USDC DEX (Smart Collateral Only)
+        updateDexBaseLimits(24, 7_500_000, 0); // fxUSD-USDC DEX (Smart Collateral Only)
+//        updateDexBaseLimits(25, 0, 0);                    // Skip DEX 25
+//        updateDexBaseLimits(26, 0, 0);                    // Skip DEX 26
+    }
+
+    // @notice Action 4: Set creation code for SmartLendingFactory
+    function action4() internal isActionSkippable(4) {
+        address SMART_LENDING_FACTORY = 0xe57227C7d5900165344b190fc7aa580bceb53B9B;
+        address SSTORE2_POINTER = 0x99a516222A64c7F5FFdC760a3bc28905140e666D;
+        address SSTORE2_DEPLOYER = 0x94a58428980291Af59adfe96844FAff088737e8F;
+
+        // Get the creation code directly from the pointer
+        bytes memory creationCode = ICodeReader(SSTORE2_DEPLOYER).readCode(SSTORE2_POINTER);
+
+        // Set the creation code in the factory
+        ISmartLendingFactory(SMART_LENDING_FACTORY).setCreationCode(creationCode);
+    }
+
+    // @notice Action 5: Transfer 500k FLUID to Team Multisig
+    function action5() internal isActionSkippable(5){
+        string[] memory targets = new string[](1);
+        bytes[] memory encodedSpells = new bytes[](1);
+
+        string
+            memory withdrawSignature = "withdraw(address,uint256,address,uint256,uint256)";
+
+        // Spell 1: Transfer INST to Team Multisig
+        {
+            uint256 FLUID_AMOUNT = 500_000 * 1e18; // 500k FLUID
+            targets[0] = "BASIC-A";
+            encodedSpells[0] = abi.encodeWithSignature(
+                withdrawSignature,
+                FLUID_ADDRESS,
+                FLUID_AMOUNT,
+                TEAM_MULTISIG,
+                0,
+                0
+            );
+        }
+
+        IDSAV2(TREASURY).cast(targets, encodedSpells, address(this));
+    }
+
+    // @notice Action 6: Delist mETH vaults
+    function action6() internal isActionSkippable(6){
+        // Delist mETH<>USDC vault
+        {
+            address vault_meth_usdc = getVaultAddress(81);
+            // Pause supply and borrow limits
+            setSupplyProtocolLimitsPaused(vault_meth_usdc, mETH_ADDRESS);
+            setBorrowProtocolLimitsPaused(vault_meth_usdc, USDC_ADDRESS);
+            // Pause user operations for both tokens
+            address[] memory supplyTokens = new address[](1);
+            address[] memory borrowTokens = new address[](1);
+            supplyTokens[0] = mETH_ADDRESS;
+            borrowTokens[0] = USDC_ADDRESS;
+            LIQUIDITY.pauseUser(vault_meth_usdc, supplyTokens, borrowTokens);
```

Comment:
no need to pause, that was already done via team multisig as guardian for all https://etherscan.io/tx/0x0c65de8a9fdd27755f358f875773cf4f9732ae79aef37cf8d16e1744612bba8b#eventlog